egs/babel_multilang/s5/local/chain2/run_tdnn.sh

```diff
@@ -405,27 +408,23 @@ fi
 
 if [ $stage -le 17 ]; then
     echo "$0: Preparing initial acoustic model"
-    if [ -f $dir/configs/init.config ]; then
-            $cuda_cmd ${dir}/log/add_first_layer.log \
-                    nnet3-init --srand=${srand} ${dir}/configs/init.raw \
-                    ${dir}/configs/final.config ${dir}/init/default.raw || exit 1
-    else
-            $cuda_cmd ${dir}/log/init_model.log \
-               nnet3-init --srand=${srand} ${dir}/configs/final.config ${dir}/init/multi.raw || exit 1
-    fi
+    $cuda_cmd ${dir}/log/init_model.log \
+           nnet3-init --srand=${srand} ${dir}/configs/final.config ${dir}/init/multi.raw || exit 1
 fi
 
 if [ $stage -le 18 ]; then
   echo "$0: Starting model training"
   steps/chain2/train.sh \
     --stage $train_stage --cmd "$cuda_cmd" \
     --multilingual-eg true \
-    --xent-regularize $xent_regularize --leaky-hmm-coefficient 0.1 \
+    --xent-regularize $xent_regularize --leaky-hmm-coefficient 0.25 --out-of-range-regularize 0.0 \
```

Comment:
Was the out-of-range-regularize 0.0 necessary?  Or a debugging thing?

Reply:
No, I don't think it is necessary to set it to 0.0. It was for debugging purposes only. I can remove it and test it again.

Reply:
I removed it and change is pushed.

Reply:
> @mrsrikanth please not the conflict!

@danpovey I don't understand, what conflict are you referring to?